preprocessors/Emilia/README.md

```diff
@@ -0,0 +1,104 @@
+## Emilia
+
+This is the official repository for the **Emilia** dataset, and the source code of the **Emilia-Pipe**. This README file will introduce the project and provide an installation guide.
+
+### Pipeline
+
+0. Standardization：Audio normalization
+1. Source Separation: Long audio -> Long audio without BGM
+2. Speaker Diarization: Get medium-length single-speaker speech data
+3. Fine-grained Segmentation by VAD: Get 3-30s single-speaker speech segments
+4. ASR: Get transcriptions of the speech segments
+5. Filtering: Get the resulting dataset
+
+## Setup Steps
+
+### 0. Prepare Environment
+
+1. Install Python and CUDA.
+2. Run the following commands to install the required packages:
+
+```bash
+conda create -y -n AudioPipeline python=3.9 
+conda activate AudioPipeline
+
+bash env.sh
+```
+
+3. Download the model files.
```

Comment:
@lixuyuan102 Hi xuyuan, could you please help add the links to all the used pre-trained models here under line 28?

Reply:
@HarryHe11 Hi Haorui, the links are added in lines 30-43.

Reply:
Thx!